src/main/java/streams/Main.java

```diff
@@ -0,0 +1,51 @@
+package streams;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
```

Comment:
Instead of declaring throws IOException, it's better to handle exceptions within the main method. This allows for graceful error handling and provides users with meaningful error messages.